doc/gitlab.nvim.txt

```diff
@@ -565,36 +607,94 @@ reviewer when checking out a new branch:
-    vim.keymap.set("n", "glaa", gitlab.add_assignee)
-    vim.keymap.set("n", "glad", gitlab.delete_assignee)
-    vim.keymap.set("n", "glra", gitlab.add_reviewer)
-    vim.keymap.set("n", "glrd", gitlab.delete_reviewer)
-    vim.keymap.set("n", "glp", gitlab.pipeline)
-    vim.keymap.set("n", "glo", gitlab.open_in_browser)
-    vim.keymap.set("n", "glM", gitlab.merge)
-    vim.keymap.set("n", "glu", gitlab.copy_mr_url)
-    vim.keymap.set("n", "glP", gitlab.publish_all_drafts)
-    vim.keymap.set("n", "glD", gitlab.toggle_draft_mode)
+    vim.keymap.set("n", "gl<c-s>", gitlab.print_settings, { desc = "Print gitlab.nvim settings"})
+<
+See |gitlab.nvim.api| for an overview of available API functions that you can
+use in your mappings.
+
+If you want to set up your own additional keybindings for the discussion tree,
+you can put them in the `$XDG_CONFIG_HOME/nvim/after/ftplugin/gitlab.lua`
+file. E.g., if you wanted to use the `j` and `k` keys to move to the next
+discussion node instead of the next line, you could add to the |ftplugin| file
+the following lines:
+>lua
+    vim.keymap.set("n", "j", [[<Cmd>call search('[] @')<CR>]], { buffer = 0, desc = "Go to next node" })
+    vim.keymap.set("n", "k", [[<Cmd>call search('[] @', 'b')<CR>]], { buffer = 0, desc = "Go to previous node" })
 <
 
+Reviewer keybindings ~
+
+Most of the keybindings `gitlab.nvim` sets are normal mode mappings, with the
+exception of `keymaps.reviewer.create_comment` and
+`keymaps.reviewer.create_suggestion` which work in both normal and visual
+mode. In normal mode, these keybindings are |operator|s that accept a |motion|
+(with an optional |count|). E.g., `c2j` will create a comment for the current
+and the next 2 lines. Similarly, `sip` will create a suggestion for the "inner
+paragraph". The operator forces |linewise| visual selection, so it works
+correctly even if the motion itself works |characterwise| (e.g., |i(| for
+selecting the inner parentheses block).
+
+The keybindings also work in visual mode, e.g., if you first want to make
+sure you are commenting on the right text segment/object, you can do `v4j` to
+visually select the current and the next 4 lines, followed by either `c` for a
+normal comment or `s` for a suggestion.
+
+To create a comment or suggestion for the current line, just duplicate the
+keybinding: `cc` and `ss`. Alternatively, you can use any motion that moves on
+that line only, e.g., `cw`, `cl`, `c$`, etc.
+
+The plugin takes advantage of the fact that the `c` and `s` normal mode
+commands modify the text, but the Diffview buffers are set as |nomodifiable|,
+so the keys are practically free for remapping. If, however, you don't like
+the default `c` and `s` keybindings, you can remap these operators to a
+different key or keys, e.g. `keymaps.reviewer.create_comment = "<leader>c"`.
+This will still work for commenting on the same line by duplicating the
+keybinding, e.g., `<leader>c<leader>c`.
```

Comment:
I'd delete this paragraph, this is explained in the previous paragraph. Or you could just add: "This same logic will apply if you remap this keybinding."

Reply:
Agree, this is unnecessary detail. Removed.